slices/redis-server.yaml

```diff
@@ -0,0 +1,32 @@
+package: redis-server
+
+essential:
+  - redis-server_copyright
+
+slices:
+  bins:
+    essential:
+      - init-system-helpers_bins
+      - redis-server_config
+      - redis-tools_bins
+      - sysvinit-utils_bins
+    contents:
+      /etc/init.d/redis-server:
```

Comment:
Thanks for these slices.

It all LGTM except this one path. This is a shell script, and just by itself, I don't think it will be functional because none of these dependencies is bringing a shell. 

If you can confirm my suspicion, and given the nature of the file, I'd propose the following two solutions. Either:
 - create a new slice (e.g. `init` or `scripts`) that depends on `redis-server_bins` and a shell. I'd only create this slice if you have a use case for it. I.e., better safe than sorry, and it's easier to inflate an SDF than it is to remove something. Or,
  - simply leave the init.d script out, if you don't need it. If ever needed, we can revisit this SDF with a proper use case.

Reply:
I followed the existing precedent that is present in other existing slices.
See:
- apparmor.yaml
- dbus.yaml
- openssh-server.yaml

Reply:
Removed.